simulariumio/data_objects/unit_data.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import logging
+from pint import UnitRegistry
+
+###############################################################################
+
+log = logging.getLogger(__name__)
+
+###############################################################################
+
+
+class UnitData:
+    magnitude: float
+    name: str
+
+    def __init__(
+        self,
+        name: str,
+        magnitude: float = 1.0,
+    ):
+        """
+        This object contains data for units
+
+        Parameters
+        ----------
+        name: str
+            unit name for values (we support this list
+            https://github.com/hgrecco/pint/blob/master/pint/default_en.txt)
+        magnitude: float (optional)
```

Comment:
I got this from pint, the Python package for units. It has "quantities" that have "magnitude "and "units". Of course I'm using "name" instead of "units" because it seemed less redundant and should help indicate that it's a string. I don't feel that strongly about it though.

Reply:
Ah I see. Up to you.

Reply:
I would have gone with `timeUnit : str` and `timeScale : float` instead of a nested struct 😁